docs/Policies/App Policies/index.md

```diff
@@ -0,0 +1,247 @@
+# App Policies
+
+The app policies section defines permissions that each application is granted in the SDL environment. This is where you would change the default permissions for each application, or add rules for specific applications. Each key in this object is associated with the `fullAppID` field for an application (or `appID` if `UseFullAppID = false` in `smartDeviceLink.ini`), and each value is a struct which contains several fields to control permissions for this app. In addition, several reserved keys exist for this section:
```

Comment:
I was about to comment and say that 2 is not several in my book but then I saw the TODO, 3 is on the edge of several, up to you.